lib/core.sh

```diff
@@ -324,7 +324,10 @@ function build_image_env(){
     _check_package git
```

Comment:
In 379 the yaourt is installed inside Junest image but in order to use yaourt during the building process it must be available in the ArchLinux host.